tools/debian/control

```diff
@@ -61,28 +61,15 @@ Provides: cockpit-ssh
 Breaks: cockpit-dashboard (<< 170.x),
  cockpit-ws (<< 181.x),
 # 233 dropped jquery.js, pages started to bundle it (commit 049e8b8dce)
- cockpit-dashboard (<< 233),
  cockpit-networkmanager (<< 233),
  cockpit-storaged (<< 233),
  cockpit-system (<< 233),
  cockpit-tests (<< 233),
-Replaces: cockpit-dashboard (<< 170.x), cockpit-ws (<< 181.x)
+Replaces: cockpit-ws (<< 181.x)
```

Comment:
These replaces/breaks must stay for a few years until 233 is in the oldest supported Debian/Ubuntu stable.

Reply:
Good point. `cockpit-dashboard` was replaced twice, so just kept one.